adr/query.py

```diff
@@ -62,6 +63,12 @@ def query_activedata(query, url):
 
 
 def load_query(name):
+    with open(os.path.join(QUERY_DIR, name + '.query')) as fh:
+        for query in yaml.load_all(fh):
+            yield query
+
+
+def updated_load_query(name):
```

Comment:
I would recommend against creating duplicate implementations in your patch, it makes things more complicated to maintain as the patch author, as well as review (it's hard for me to see what was modified this way). Just make the change (even if it means some of the recipes are temporarily broken). It's the job of version control to keep track of a historical record, so if we ever decide that the change is a bad idea, we can retrieve the old implementation from vcs.